types/web3/web3-tests.ts

```diff
@@ -125,6 +139,95 @@ web3.eth.personal.unlockAccount(
 //
 // web3.eth.abi
 // --------------------------------------------------------------------------
+const myContractOldAbiStandard: ABIDefinition[] = [
+    {
+        constant: true,
+        payable: false,
+        inputs: [
+            {
+                name: "paramAddress",
+                type: "address"
+            }
+        ],
+        name: "function1",
+        outputs: [],
+        type: "function"
+    }, {
+        constant: true,
+        payable: false,
+        inputs: [],
+        name: "removeOwner",
+        outputs: [
+            {
+                name: "paramAddress",
+                type: "address"
+            }
+        ],
+        type: "function"
+    }, {
+        constant: false,
+        payable: false,
+        inputs: [],
+        name: "removeOwner",
+        outputs: [],
+        type: "function"
+    }, {
+        constant: false,
+        payable: true,
+        inputs: [],
+        name: "removeOwner",
+        outputs: [],
+        type: "function"
+    },
+];
+const myContractOldAbi = new web3.eth.Contract(myContractOldAbiStandard);
+
+const myContractNewAbiStandard: ABIDefinition[] = [
+    {
+        constant: true,
+        payable: false,
+        stateMutability: "view",
+        inputs: [
+            {
+                name: "paramAddress",
+                type: "address"
+            }
+        ],
+        name: "function1",
+        outputs: [],
+        type: "function"
+    }, {
+        constant: true,
+        payable: false,
+        stateMutability: "pure",
+        inputs: [],
+        name: "removeOwner",
+        outputs: [
+            {
+                name: "paramAddress",
+                type: "address"
+            }
+        ],
+        type: "function"
+    }, {
+        constant: false,
+        payable: false,
+        stateMutability: "nonpayable",
+        inputs: [],
+        name: "removeOwner",
+        outputs: [],
+        type: "function"
+    }, {
+        constant: false,
+        payable: true,
+        stateMutability: "payable",
+        inputs: [],
+        name: "removeOwner",
+        outputs: [],
+        type: "function"
+    },
+];
+const myContractNewAbi = new web3.eth.Contract(myContractNewAbiStandard);
```

Comment:
Thank you. Is it possible to create a second test file for this?

Reply:
The only file tested (linted) by `npm run test web3` is `web3-tests.ts`. Therefore I have added `test/abi-tests.ts`, containing an example of old and new abi standard.